client/llb/source.go

```diff
@@ -452,6 +452,45 @@ func Differ(t DiffType, required bool) LocalOption {
 	})
 }
 
+func OCILayout(name string, opts ...OCILayoutOption) State {
```

Comment:
This signature is wrong https://github.com/moby/buildkit/pull/2827#discussion_r862207587

Reply:
I'll check.

Reply:
IS it? I looked at your comment above, you suggested:

```go
func OCILayout(contentStoreID, digest string, opts ...OCILayoutOptions)
```

But in the end, all it does is return a call to `NewSource`, whose signature is:

```go
func NewSource(id string, attrs map[string]string, c Constraints) *SourceOp {
```

So it is just going to combine the contentStoreID and hash anyways, isn't it? What do I gain by:

```go
func OCILayout(contentStoreID, digest string, opts ...OCILayoutOptions)
...
	source := NewSource("oci-layout://"+contentStoreID+"@"+digest, attrs, gi.Constraints)
```

Over just:

```go
func OCILayout(ref string, opts ...OCILayoutOptions)
...
	source := NewSource("oci-layout://"+ref, attrs, gi.Constraints)
```

?